.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+style = defaultWithAlign
```

Comment:
FTR, I *personally* don't like this option, but I see value in compromising on the formatting options. After all, we all have our opinions.
  
This is how you do it in Intellij http://stackoverflow.com/questions/19935317/ignore-whitespace-changes-in-intellij-changebars (see the second answer). With git, you do it with `git diff -w`.